GameServer.cpp

```diff
@@ -393,7 +394,6 @@ void GameServer::ProcessRequestExternalAddress(Packet& packet, sockaddr_in& from
 
 void GameServer::DoTimedUpdates()

```

Comment:
DoTimedUpdates is too large a function. I'd like to refactor it in the future into 2 or more smaller functions.

Reply:
It is definitely more deeply nested than I'd like. Some refactoring seems to be in order.